msteams-platform/concepts/design/plan-responsive-tabs-for-teams-mobile.md

```diff
@@ -122,7 +122,7 @@ In the third stage, you can enable users' experience with the following activiti
 * Add and modify different categories, such as **Proposed**, **Active**, and **Closed**.
 * Configure the tasks for comments, attachments, and other complex features.
 
-:::image type="content" source="../../assets/images/app-fundamentals/taskboarding-apps-enablement.png" alt-text="Enable the user experience by adding projects and boards.":::
+:::image type="content" source="../../assets/images/app-fundamentals/taskboarding-apps-enablement.png" alt-text="The screenshot shows the steps to enable the user experience by adding projects and boards.":::
```

Comment:
The screenshot doesn't show the steps. It only shows an example.

Reply:
resolved